Dockerfile

```diff
@@ -5,5 +5,8 @@ ADD --chown=www-data:www-data ./nlw-madoc-theme /srv/omeka/themes/nlw-madoc-them
 
 # Add custom translations
 ADD --chown=www-data:www-data ./translations/s/ /srv/omeka/translations/s/
+ADD --chown=www-data:www-data ./translations/omeka/cy.mo /srv/omeka/application/language/cy.mo
+ADD --chown=www-data:www-data ./translations/madoc/cy.mo /srv/omeka/application/madoc/cy.mo
```

Comment:
```suggestion
ADD --chown=www-data:www-data ./translations/madoc/cy.mo /srv/omeka/translations/madoc/cy.mo
```

Reply:
The Madoc translations are stored in `./translations/madoc/*.mo`

Reply:
ah ok, that makes sense. I copied the path from the Admin "where to add translations"  for each file. I did wonder why it was different to the other one in custom translations section.